test/cli_logs_test.go

```diff
@@ -251,7 +251,9 @@ func (suite *PouchLogsSuite) TestSetLogPathInDaemon(c *check.C) {
 	logRootDir := "/tmp/TestSetLogsPathInDaemon"
 	defer os.RemoveAll(logRootDir)
 
-	dcfg, err := StartDefaultDaemon("--log-opt", fmt.Sprintf("root-dir=%s", logRootDir))
+	dcfg, err := StartDefaultDaemon(map[string]interface{}{
+		"--log-opt": fmt.Sprintf("root-dir=%s", logRootDir),
```

Comment:
This need use args instead of config field? Or use `log-opt` instead of `--log-opt`? I'm not sure 😢